sbin/build.sh

```diff
@@ -962,7 +962,8 @@ addInfoToReleaseFile() {
 
 addHeapSize() { # Adds an identifier for heap size on OpenJ9 builds
   local heapSize=""
-  if [[ $($JAVA_LOC -version 2>&1 | grep 'Compressed References') ]]; then
+  local architecture=$($JAVA_LOC -XshowSettings:properties -version 2>&1 | grep 'os.arch' | sed 's/^.*= //' | tr -d '\r')
```

Comment:
Can we add an issue comment link for why this is necessary?

Reply:
```
addHeapSize() { # Adds an identifier for heap size on OpenJ9 builds
  local heapSize=""
  local architecture=$($JAVA_LOC -XshowSettings:properties -version 2>&1 | grep 'os.arch' | sed 's/^.*= //' | tr -d '\r') # Heap size must be standard for x86 builds (openjdk-build/2412)
```

Fine by you?

Reply:
Could maybe add a link to https://github.com/AdoptOpenJDK/openjdk-build/issues/2412 but that's ambiguous and is fine as is 🙂